### PR TITLE
move UnsizedItemStackSet to net.minecraft.item, fixes #277

### LINKED ISSUE #277: `UnsizedItemStackSet` is in `net.minecraft.unmapped` package

### PR DESCRIPTION
**Pull request by eristhea**
I also mapped its two methods

### CODE REVIEW

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/item/UnsizedItemStackSet.mapping [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/unmapped/C_voakmxnc net/minecraft/item/UnsizedItemStackSet
+	FIELD f_vjorabmg hashStrategy Lit/unimi/dsi/fastutil/Hash$Strategy;
```
Comment (OroArmor) [inline mappings/net/minecraft/item/UnsizedItemStackSet.mapping]:
```suggestion
	FIELD f_vjorabmg HASH_STRATEGY Lit/unimi/dsi/fastutil/Hash$Strategy;
```

**Review by OroArmor — CHANGES_REQUESTED** [review]

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/item/UnsizedItemStackSet.mapping [hunk 2]
```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/unmapped/C_voakmxnc net/minecraft/item/UnsizedItemStackSet
+	FIELD f_vjorabmg HASH_STRATEGY Lit/unimi/dsi/fastutil/Hash$Strategy;
+	METHOD m_llcvfzzg hashCode (Lnet/minecraft/unmapped/C_sddaxwyk;)I
+	METHOD m_nnsofwir get ()Ljava/util/Set;
+	CLASS C_sansacgo
+		METHOD equals (Ljava/lang/Object;Ljava/lang/Object;)Z
+			ARG 1 stack
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/item/UnsizedItemStackSet.mapping]:
```suggestion
			ARG 1 stack1
```
If there's a `stack2` it seems like the first should be `stack1`?

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/item/UnsizedItemStackSet.mapping (hunk 2 above)
Comment (eristhea) [reply mappings/net/minecraft/item/UnsizedItemStackSet.mapping]: that's carried over from the old class, I'll look and see if I can name them something more descriptive

Comment (eristhea) [reply mappings/net/minecraft/item/UnsizedItemStackSet.mapping]: oh wait never mind that's an equals method

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by IotaBread — APPROVED** [review]

**Pull request merged**